oda_api/cli.py

```diff
@@ -41,9 +41,8 @@ def cli(obj, debug=False, dispatcher_url=None, test_connection=False, wait=True,
 @click.option("-i", "--instrument", default=None)
 @click.option("-p", "--product", default=None)
 @click.option("-a", "--argument", default=None, multiple=True)
-@click.option("-T", "--discover-token", "_discover_token", is_flag=True)
```

Comment:
the option is still useful since sometimes I do not want it to implicitly find the token. could you maybe change it to something like "--token-discovery-methods" and assume default to discover with those methods?

Reply:
I assume you mean that it defaults to the method used by the `discover_token` function ?

Reply:
so now, when is not passed, the token will be discovered in the "standard" way, otherwise it will be searched and used